res/openvpn-gui-res-no.rc

```diff
@@ -187,9 +187,9 @@ BEGIN
     LTEXT "OpenVPN GUI v" PACKAGE_VERSION " - Ett Windows-Grensesnitt mot OpenVPN\n" \
           "Copyright (C) 2004-2005 Mathias Sundman <info@openvpn.se>\n" \
```

Comment:
Copyright has to be updated in all language files -- let's do that in a separate patch as some change sin wording may be required.  See also https://github.com/OpenVPN/openvpn-gui/issues/81